doc/clap.txt

```diff
@@ -411,12 +421,12 @@ g:clap_provider_grep_blink                         *g:clap_provider_grep_blink*
 g:clap_provider_grep_opts                          *g:clap_provider_grep_opts*
 
   Type: |String|
-  Default: `''`
+  Default: `'-H --no-heading --vimgrep --smart-case --hidden -g "!.git/"'`
 
-  Set this to control the extra arguments that get passed to the grep tool.
-  Setting `'--hidden -g "!.git/"'` for example would enable grepping hidden
```

Comment:
What I actually mean is to keep this sentence but replacing `'--hidden -g "!.git/"'` with `'-H --no-heading --vimgrep --smart-case --hidden -g "!.git/"'`.

Reply:
Ohhhhh, completely misunderstood you! Yeah that makes totally sense. Fixed!